### ISSUE TITLE
Old names tools on Wiki pages

### ISSUE THREAD

**Issue by GermanAizek** [rizinorg/rizin#4125]
Hi,
Is there any information in wiki or documentation that shows migrate history old radare2 names and new rizin. Everyone is used to old r2 names, for example, go to the wiki page https://book.rizin.re/tools/rz-diff/intro.html but it is not indicated anywhere that old name this tool is "radiff2", it is so inconvenient when there are few tutorials on rizin, but I want to perform the same action using the radare2 tutorial.

- Example tutorial: https://rderik.com/blog/using-radare2-to-patch-a-binary/

As result, beginners use r2, or need to come up with alias for package names.

- Example wiki page:

![image](https://github.com/rizinorg/rizin/assets/21138600/12af6439-6ed2-438a-aeae-823f253db8d5)

**Comment by XVilka** [comment]
@GermanAizek, good point. Though the difference grows with every release, commands and options also begin to differ. I would say "Migration from Radare2" would be better suited, e.g., in the "Migration" chapter of the book: https://book.rizin.re/debugger/migration.html

Thus, moving this issue to the book repo.
